source/extensions/filters/http/aws_lambda/config.h

```diff
@@ -97,6 +98,10 @@ class AWSLambdaConfigImpl
       SharedAWSLambdaProtocolExtensionConfig ext_cfg,
       StsConnectionPool::Context::Callbacks *callbacks) const override;
 
+    bool ShouldPropagateOrigin() const override{
+      return propagate_original_routing_;
```

Comment:
nit, let's have the method name match the variable name here (I prefer `propagate_original_routing_`)

Reply:
Also typically cpp methods start with lower case